components/formats-gpl/src/loci/formats/in/LEOReader.java

```diff
@@ -88,25 +90,32 @@ protected void initStandardMetadata() throws FormatException, IOException {
 
     String tag = ifds.get(0).getIFDTextValue(LEO_TAG);
     String[] lines = tag.split("\n");
-
     date = "";
 
-    for (int line=10; line<lines.length; line++) {
-      if (lines[line].equals("clock")) {
-        date += lines[++line];
-      }
-      else if (lines[line].equals("date")) {
-        date += " " + lines[++line];
-      }
-    }
-
     if (getMetadataOptions().getMetadataLevel() != MetadataLevel.MINIMUM) {
-      // physical sizes stored in meters
-      xSize = Double.parseDouble(lines[3]) * 1000000;
+      for (int line=36; line<lines.length; line++) {
+        if (lines[line].equals("AP_IMAGE_PIXEL_SIZE")) {
+          // pixel size is stored in nm, converted now to micrometers
+          double xSize = Double.parseDouble(lines[++line].split("\\s+=\\s+")[1].replace(" nm","e-03"));
```

Comment:
As the variables xSize etc have already been declared as instance variables then you can simply use:
`xSize = Double.parseDouble(lines[++line].split("\\s+=\\s+")[1].replace(" nm","e-03"));`

Declaring it again creates a local variable which is not used outside of the block and as such results in the default values still appearing. Making that small change sees the values being parsed and populated correctly with the sample files from https://keybase.pub/dmankus/